onyo/lib/consts.py

```diff
@@ -1,20 +1,16 @@
 from collections import UserDict
 from typing import TYPE_CHECKING
+
+from onyo.lib.pseudokeys import PSEUDOKEY_ALIASES
 if TYPE_CHECKING:
     from typing import Literal
     sort_t = Literal['ascending', 'descending']
 
 
-PSEUDO_KEYS = ['path']
-r"""Key names that are addressable but not in asset content.
-
-All ``PSEUDO_KEYS`` are reserved.
-
-See Also
---------
-RESERVED_KEYS
-"""
-RESERVED_KEYS = ['directory', 'is_asset_directory', 'template']
+RESERVED_KEYS = ['template', 'onyo'] + list(PSEUDOKEY_ALIASES.keys())
```

Comment:
More "talking out loud" rather than anything else.

I'm not sure if aliases should be reserved. I just think they should be translated.

Though I see this is a nice way to make sure they don't somehow leak into the asset content un-translated.

I'm split.

Part of me wants them to just translate on the way in, and fsck does the actual checks.

Reply:
> However, the way RESERVED_KEYS is used ATM, this all needs to be in there. So, this isn't meant to be the final form, but the intermediate definition that allows this PR to not break shit. We reserved these keys before this PR and after. Just not at the end of the series.

Cool. I'm fine with that then. Last time I looked seriously into this, I found just how intertwined the logic is. So +1 on doing this piece by piece.